Dave-mash/StackOverflow-lite
# [finishes #162573442] Added an answers section to each question

# What does this PR do?
Adds a section for viewing answers to each questions
# How should this be manually tested?
* Clone this repository on your machine:
$ git clone https://github.com/Dave-mash/stackOverflow-lite.git
* CD into the project folder on your machine
$ cd stackOverflow-lite
* Check out the branch for this PR
$ git branch ft-user-login-162573442
* Open the index.html file on your browser
# Pivotal Tracker story associated with this 
PRStory-ID #162573442
Screenshot::
![view](https://user-images.githubusercontent.com/41325679/49895494-9df28680-fe61-11e8-8764-2df3a8fc7535.png)
